packages/itwinui-react/src/core/Tooltip/Tooltip.test.tsx

```diff
@@ -20,6 +20,7 @@ it('should toggle the visibility of tooltip on hover', () => {
 
   const tooltip = getByText('some text');
   expect(tooltip).not.toBeVisible();
+  expect(tooltip).toHaveAttribute('popover');
```

Comment:
Should it also check that the attribute value is `manual`?

Reply:
The value is not important here. The `popover` attribute is what really matters. Every value of `popover` puts it in the top layer.

Reply:
optional: Since there are some differences between `auto` and `manual` ([docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/popover#value)), I thought it might not hurt to confirm that `manual` is selected.

Reply:
Sure, done ([`1ce6774`](https://github.com/iTwin/iTwinUI/pull/2060/commits/1ce67744f52effe5c68739f20977faeb40c6a4ad)). But I'll reemphasize that it's _not_ important, it's almost like testing implementation details. And improving unit tests is a waste of everyone's time in most cases.